src/index.ts

```diff
@@ -566,4 +566,12 @@ if (!amMainInstance) {
             app.quit();
         }
     });
+}
+
+
+export function autoHideMenuBar(status: boolean){
+    windows.forEach((window) =>
+        window.setAutoHideMenuBar(status || false)
```

Comment:
I don't think you need `|| false` here: TypeScript will enforce that the status is a boolean (so not undefined or null, always just `true` or `false`) so AFAICT this isn't doing anything.